assignments/closure.js

```diff
@@ -14,7 +25,21 @@ const counter = () => {
 
 // ==== Challenge 3: Create a counter function with an object that can increment and decrement ====
 const counterFactory = () => {
+  let obj = {"count": 0};
+  obj["increment"] = function(){
+    return ++this.count;
+  }
+  obj["decrement"] = function(){
+    return --this.count;
+  }
+  return obj;
```

Comment:
This gets the job done, but is a little lengthy and isn't taking full advantage of closures. You could've done it this way as well:
```javascript
const counterFactory = () => {
  let count = 0;
  return {
    increment: () => ++count;
    decrement: () => --count;
  };
}
```